projects/plugins/boost/app/lib/Boost_Health.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Lib;
+
+use Automattic\Jetpack_Boost\Lib\Critical_CSS\Critical_CSS_State;
+use Automattic\Jetpack_Boost\Modules\Optimizations\Cloud_CSS\Cloud_CSS;
+
+class Boost_Health {
+
+	private $issues = array();
+
+	public function __construct() {
+		if ( self::critical_css_needs_regeneration() ) {
+			$this->issues[] = 'Outdated Critical CSS';
+		}
+
+		if ( self::critical_css_has_errors() ) {
+			$this->issues[] = 'Failed to generate Critical CSS';
```

Comment:
untranslated strings.

Reply:
😅 Fixed in [0dec583](https://github.com/Automattic/jetpack/pull/31101/commits/0dec5839e3a1c469cb1020436b2df42b6fea1989).